packages/icons/src/Spinner/index.js

```diff
@@ -24,9 +24,8 @@ const Spinner = ({
   ...rest
 }) => (
   <SVG className={className} viewBox="-25 -25 50 50" preserveAspectRatio="xMidYMid meet" title={title} fill={fill} {...rest}>
-    {Array(12).fill(1).map((el, i) =>
-      /* eslint-disable-next-line react/no-array-index-key */
-      <Rect fill={fill} width="12" height="5" rx="2.5" ry="2.5" style={{ animationDelay: `${i * 83}ms` }} transform={`rotate(${i * 30}, 0, 2) translate(10 0)`} opacity="0" key={i} />)}
+    {/* eslint-disable-next-line react/no-array-index-key */}
+    {Array(12).fill(1).map((el, i) => <Rect fill={fill} width="12" height="5" rx="2.5" ry="2.5" style={{ animationDelay: `${i * 83}ms` }} transform={`rotate(${i * 30}, 0, 2) translate(10 0)`} opacity="0" key={i} />)}
```

Comment:
You spin me right round baby right round like a record baby, right round, right round.